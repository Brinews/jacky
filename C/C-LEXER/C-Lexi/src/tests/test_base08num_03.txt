18x08
0x08
