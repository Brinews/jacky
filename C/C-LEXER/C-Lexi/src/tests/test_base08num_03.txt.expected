{18 , NUM , 1}
{x08 , ID , 1}
{0x08 , BASE08NUM , 2}
{ , END_OF_FILE , 4}
